Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "pg_doorman"
 version = "1.7.8"
 edition = "2021"
+rust-version = "1.85.0"
```

Comment:
We aim to package this project with support for multiple Rust versions, as it is occasionally necessary to build it using the Rust compiler provided by the specific distribution. This approach ensures flexibility and compatibility across diverse environments.

Reply:
I've downgraded MSRV to 1.70 (this parameter is only the minimal version)

Still kept `rust-toolchain.toml` referencing `1.85` since duriing development it is preferable to use the latest toolchain

Reply:
Oh, cool, it helped find an incompatibility:

![image](https://github.com/user-attachments/assets/2c38af93-c90f-4442-a072-f386f766a06c)